mreg/api/v1/tests/tests.py

```diff
@@ -557,6 +558,36 @@ def test_hosts_post_case_insenstive_201_created(self):
         response = self.assert_post_and_201('/hosts/', data)
         self.assertEqual(response['Location'], '/api/v1/hosts/%s' % self.post_data['name'])
 
+    def prepare_allocator_data(self, allocation_method):
+        network = '10.0.0.0/24'
+        Network.objects.create(network=network)
+        data = self.post_data.copy()
+        data['allocation_method'] = allocation_method
+        data['network'] = network
+        del data['ipaddress']
+        return data
+
+    def assert_successful_allocation(self, allocation_method):
+        data = self.prepare_allocator_data(allocation_method)
+        response = self.assert_post_and_201('/hosts/', data)
+        self.assertEqual(response['Location'], f'/api/v1/hosts/{self.post_data["name"]}')
+        # TODO: Implement validation for IP address allocation
```

Comment:
What is meant by this? That we ensure the IP that was allocated is valid IP in general? Is an IP in the given network? Doesn't overlap with an existing host's IP?

Reply:
I'd love to be able to validate that we got the address we expected, but I don't really know how to do that with random allocation. :(